src/PHASE_3_RANSAC_FILTERING/ransac_filter.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 16 14:46:18 2018
+
+@author: mmacferrin
+
+Code using OpenCV, from "Learn OpenCV" tutorial: https://www.learnopencv.com/image-alignment-feature-based-using-opencv-c-python/
+"""
+
+#######################################################################################
+## Code for importing the Utilities directory
+import os,sys,inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+utilities_dir = os.path.join(parentdir, "UTILITIES")
+sys.path.insert(0,utilities_dir)
+#######################################################################################=
+
+
+from Matches_Reader import ASIFT_Matches
+from CSV_writer import write_csv
+
+# External open CV v3 or higher
+import argparse
+import cv2
+import numpy
+from osgeo import gdal, gdalconst
+
+def find_nodata_matches(keypoints_array, img_filename, img_nodata=None):
+    '''Take the keypoints given for this image, find the nodata value, and search the image to find all points that are adjacent to nodata values.
```

Comment:
I would suggest keeping the comments short as much as possible.
```suggestion
    '''Take the keypoints given for this image, find the nodata value, and search the image to find all points that are adjacent to nodata values.
```

Reply:
Aymen, shorter comments are fine with me. I was just putting together a doc-string for the function to use for documentation later, but obviously it can be modified however y'all feel is the best way to present it.

Reply:
I would like to correct that a bit. @mmacferrin I agree the doc string should be there to be used in the documentation later, and thank you for being ahead of the curve there! I think what @AymenFJA wanted to say is try and keep each line around 80 to 100 characters.

It usually makes the code a bit easier to read!

Reply:
Ah yes, of course. And I agree. I have a bad habit of using my widescreen monitor and not paying much attention to the comment width, which I always notice later when opening it on my laptop. I can take care to fix that and shorten the comment widths. 80 chars is pretty standard.